.github/workflows/gradle.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
 
     outputs:
-      publish: ${{ steps.publish_vars.outputs.release != 'true' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/4.')) }}
+      publish: ${{ steps.publish_vars.outputs.release != 'true' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/issue-3849') }}
```

Comment:
This checks are written to prevent unwanted releases. Now we have a new "release" that will be picked as latest-gratest

Reply:
this is temporary, we'll remove this before merging.